Microsoft.HealthVault/Connection/SessionCredentialClientBase.cs

```diff
@@ -59,6 +59,8 @@ private SessionCredential GetSessionCredential(HealthServiceResponseData respons
             }
 
             SessionCredential sessionCredential = this.GetAuthenticationToken(responseData.InfoNavigator);
+            // Based on access token default lifetime: https://docs.microsoft.com/en-us/azure/active-directory/active-directory-configurable-token-lifetimes
```

Comment:
The references in Platform go into a Microsoft Passport library for auth, I'll get clarification from the core team

---
In reply to: [115798063](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115798063) [](ancestors = 115798063)